lib/rules/no-unused-vars.js

```diff
@@ -92,7 +103,8 @@ module.exports = {
             vars: "all",
             args: "after-used",
             ignoreRestSiblings: false,
-            caughtErrors: "all"
+            caughtErrors: "all",
+            reportUsedIgnorePattern: false,
```

Comment:
```suggestion
            reportUsedIgnorePattern: false
```

To fix the lint error.